server.js

```diff
@@ -1,4 +1,4 @@
-// TASK 1: Setting up server
```

Comment:
The comment for task 1 is misspelled. It should be 'TASK 1' instead of 'TATASKSK 1'.